cli/.js/src/main/scala/aqua/ArgOpts.scala

```diff
@@ -101,7 +101,8 @@ object ArgOpts {
 
     data match {
       case None if vars.nonEmpty =>
-        invalidNec("Function have non-literals, so, data should be presented")
+        // TODO: replace someArguments with actual argument names that where present in the function call
+        invalidNec("Function call has '(someArguments)' but you didn't provide data or data-path argument when you called 'aqua run'. If you want to pass JSON data as an argument please provide data or data-path argument.")
```

Comment:
--data or --data-path?
@DieMyst could you please print a comma-separated list of vars here

Reply:
plz separate into several lines so it's readable in console

Reply:
There's a well-known notion of undefined variables. I'd write it like this:
```
function(a, b, c, d)
               ^^^
Undefined variables: a, b. You can provide them via --data or --data-path
```